docs/plugins/owners.md

```diff
@@ -10,12 +10,12 @@
 
 基于目前 TiDB 的 SIG 的设计，将权限划分如下：
 
-- committers（**可以使用 /merge 命令**）
+- committers（**可以合并 PR**）
   - maintainers
   - techLeaders
   - coLeaders
   - committers
-- reviewers (**可以使用 /lgtm 命令**)
+- reviewers (**可以点赞 PR**)
```

Comment:
Maybe it's not a good idea to use `点赞`? Because many contributors who are new to the community don't know what the term means.  We also don't have explanations for these proper nouns.(Maybe there is or should be one in the contribution guide we are working on?) Maybe it could be `批准`?

Reply:
I'd prefer `赞成`